db/patch.js

```diff
@@ -4,4 +4,4 @@
 
 // The expected patch level of the database. Update if you add a new
 // patch in the schema/ directory.
-module.exports.level = 7
```

Comment:
Or we leave the patches as-is like we had before, but we create the `train-xxx.sql` file for actual releases. Yes, this means dev/prod would be slightly different but would work. How does that sound?